### ISSUE TITLE
Adding manifest, icons and working on UI using Bootstrap.

### ISSUE THREAD

**Issue by LoneWolf36** [UdacityMobileWebScholarship/just-notes#17]
#### I'm opening this issue because: I want to setup the manifest file and work on UI using Bootstrap.

  - Other (_see below for feature requests_):

 - [ ] `node -v` prints: v9.11.1
 - [ ] `npm -v` prints: 6.0.0
 - [ ] Windows, OS X/macOS, or Linux?: Linux
 - [ ] Browser: Chrome

**Comment by Nagendra1421** [comment]
Manifest is already implemented on client side, you can find it in public folder

**Comment by LoneWolf36** [comment]
@skyshader I was working on it : )
Allow me to continue on it?

**Comment by whimsicaldreamer** [comment]
@skyshader I have added favicons to the `home-screen` branch already.

**Comment by ManajitPal** [comment]
I would suggest working on UI without using bootstrap. Use flexbox/CSS grids instead. Check the `ayan-home-screen` branch for proper structuring of all the UI stuff. Also, use the variables defined in `dark.css` to add colors to the project instead of defining colors in your CSS. Cheers! 👍

**Comment by vipinkhushu** [comment]
Hi @LoneWolf36 ,
Is there some bootstrap task, i am interested.

**Comment by LoneWolf36** [comment]
Yes there is, there's a ayan-home-screen branch in which there's the UI development going, you could contribute there.

**Comment by whimsicaldreamer** [comment]
@vipinkhushu @LoneWolf36 in `ayan-home-screen` branch we are working without bootstrap.

**Comment by vipinkhushu** [comment]
@LoneWolf36 @whimsicaldreamer Working without bootstrap will be fine for me..

**Comment by Ankk98** [comment]
Is this issue resolved or any work is remaining?
I am interested in contributing.
